### DISCUSSION TITLE
使用paddleocr出现INFO: During the training process, after the 0th iteration, an evaluation is run every 2000 iterations 不运行

### DISCUSSION THREAD

**Discussion by AspikeP** [PaddlePaddle/PaddleOCR#13257] [Q&A]
1.使用cpu版本
2.测试paddleocr 2.7与main版本均出现上述问题
3.测试v3与v4训练模型均出现上述问题
4.更改batchsize和num也没有效果
5.det训练的时候 output未生成文件，rec训练生成文件并打印ppocr INFO: save model in ./output/rec_ppocr_v4\latest
请问是什么问题？

**Comment by UserWangZz** [comment]
CPU版本，会不会是速度太慢了？图片大小如何呢？

**Reply by AspikeP** [reply]
我是先做一次粗定位后在进行识别 所以图像都比较小 100*300左右的大小或者300*100 图像规格不统一，目前看rec可以正常训练，但是det还是无法训练

**Reply by UserWangZz** [reply]
可以看看config文件中对图像resize步骤会不会放大了

**Reply by AspikeP** [reply]
好起来了 原来是640*640 我改成了320*320可以了 就是有些慢

**Comment by GreatV** [comment]
安装下 paddle 3.0 beta 版本，再试试。

**Reply by AspikeP** [reply]
感谢感谢 刚刚测试main版本改了一下图像尺寸 跑起来了 就是慢一些
